Dockerfile

```diff
@@ -16,13 +16,36 @@ COPY controllers/ controllers/
 COPY pkg/ pkg/
 COPY bindata/manifests/ bindata/manifests/
 
+# Install libxdp and libbpf headers
+ENV TOOLS_VERSION=1.2.6
+ENV XDPTOOLS_SRC=/workspace/xdp-tools-$TOOLS_VERSION
+# Packages to install via APT for building.
+ENV BUILD_DEPS=" \
+    clang llvm libelf-dev libpcap-dev gcc-multilib build-essential m4 \
+    unzip \
+    wget \
+    libbpf-dev \
+"
+# install dependencies required for bulding libxdp lib
+RUN apt-get update && apt -y install \
+    ${BUILD_DEPS} && \
+    rm -rf /var/lib/apt/lists/*
+
+# Download libxdp source
+RUN wget -O xdp-tools.tar.gz "https://github.com/xdp-project/xdp-tools/releases/download/v${TOOLS_VERSION}/xdp-tools-${TOOLS_VERSION}.tar.gz" \
+    && tar xvfz xdp-tools.tar.gz \
+    && rm -f xdp-tools.tar.gz
+
+RUN cd $XDPTOOLS_SRC; ./configure; make install; cd $WORKDIR
+
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

Comment:
qq: only the daemon attached/detaches eBPF programs. Why does the operator need a libbpf / libxdp dependency?